src/game

```diff
@@ -8,7 +8,17 @@ import typer
 import moreorless
 
 app = typer.Typer()
-SCOREBOARD_PATH = "scoreboard.json"
+SCOREBOARD_PATH = "src/scoreboard.json"
```

Comment:
Better use a path outside of the project. like /tmp/more_or_less/local_scoreboard.json

Reply:
It's fix now by replacing the path by a fix path outside the game